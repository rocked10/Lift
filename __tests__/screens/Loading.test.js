import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Loading from '../../screens/loading';

describe('Testing Loading Screen', () => {
    const wrapper = renderer.create(<Loading />);

    it('Should render', () => {
        expect(wrapper.toJSON()).toMatchSnapshot();
    });
});