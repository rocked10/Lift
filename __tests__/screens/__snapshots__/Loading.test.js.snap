// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Loading Screen Should render 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#0080ff"
    hidesWhenStopped={true}
    size="large"
  />
</View>
`;
